NewForce-Cohort-6/tabloidfullstack-stuffing
# SB/4-View-Current-Users-Posts

# Description

Functionality added for a user to view their list of posts.

# Testing Instructions

`git checkout SB/4-View-Current-Users-Posts`

1. Login with any user
2. Navigate to My Posts via navbar
3. You should see a list of the currently logged in user's posts ordered by create date descending with Title and Author displayed
4. And the my posts list should contain both approved and unapproved posts
5. The title link should take you to that post's details page

NOTE: Categories will be added to list once backend API dependency is complete (Robert)

# Checklist:
